### PR TITLE
chore: set batch memory to 120G in dev for wmg

### PR DESCRIPTION
**Pull request by ebezzi**
### Reviewers
**Functional:** 

**Readability:** 

---

## Changes
- add
- remove
- modify

### CODE REVIEW

File: .happy/terraform/envs/dev/main.tf [hunk 1]
```diff
@@ -13,7 +13,7 @@ module stack {
   frontend_url                 = "https://cellxgene.dev.single-cell.czi.technology"
   backend_url                  = "https://api.cellxgene.dev.single-cell.czi.technology"
   stack_prefix                 = ""
-  batch_container_memory_limit = 60000
+  batch_container_memory_limit = 120000
```
Comment (atolopko-czi) [inline .happy/terraform/envs/dev/main.tf:16]: Usually prod resources need to be larger than dev!  It feels odd having to give in to dev's messy data state in this way.  Maybe comment why this is larger than prod? 😬

**Review by atolopko-czi — APPROVED** [review]
Comment suggestion, but if we have do it, we have to do it.

File: .happy/terraform/envs/dev/main.tf (hunk 1 above)
Comment (ebezzi) [reply .happy/terraform/envs/dev/main.tf:16]: Actually, prod has 300G!

Comment (atolopko-czi) [reply .happy/terraform/envs/dev/main.tf:16]: Ha, ok! I feel a little better about this then. 😆

**Pull request merged**